rmgpy/tools/simulateTest.py

```diff
@@ -74,6 +76,23 @@ def test_liquid(self):
         shutil.rmtree(os.path.join(folder, 'solver'))
         os.remove(os.path.join(folder, 'simulate.log'))
 
+    def test_mbSampled(self):
+        """Test that we can simulate an MBSampledReactor"""
+        folder = os.path.join(os.path.dirname(rmgpy.__file__), 'tools/data/sim/mbSampled')
```

Comment:
Need to join tools, data, sim, and mbSampled. Can't use `/`

Reply:
Thanks for keeping up with Windows support 😉

Reply:
RMG-Py: potentially Windows compatible since 2019 :)